src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -34,6 +35,67 @@ public void Deconstruct(out int a, out string b)
 
             var comp = CompileAndVerify(source, expectedOutput: "1 hello", parseOptions: TestOptions.Regular.WithTuplesFeature());
             comp.VerifyDiagnostics();
+            comp.VerifyIL("C.Main", @"
+{
+  // Code size       40 (0x28)
+  .maxstack  3
+  .locals init (string V_0, //y
+                int V_1,
+                string V_2)
+  IL_0000:  newobj     ""C..ctor()""
+  IL_0005:  ldloca.s   V_1
+  IL_0007:  ldloca.s   V_2
+  IL_0009:  call       ""void C.Deconstruct(out int, out string)""
+  IL_000e:  ldloc.1
+  IL_000f:  conv.i8
+  IL_0010:  ldloc.2
+  IL_0011:  stloc.0
+  IL_0012:  box        ""long""
+  IL_0017:  ldstr      "" ""
+  IL_001c:  ldloc.0
+  IL_001d:  call       ""string string.Concat(object, object, object)""
+  IL_0022:  call       ""void System.Console.WriteLine(string)""
+  IL_0027:  ret
+}");
+        }
+
+        [Fact]
+        public void DeconstructIL()
```

Comment:
Yes, it is now that I added VerifyIL to the test above. I'll remove the redundant one.